### PR TITLE
LDAF-341 Create Top Tier pages

### PR DESCRIPTION
**Pull request by LouisFettet**
Jira ticket: [LDAF-341](https://ldaf.atlassian.net/browse/LDAF-341)

## Proposed changes

Adds a rough implementation of the Top Tier pages, e.g. `/animals`, `/plants`, etc. Highlights are:

- Render the Top Tier pages with a hero image, title, subheading, description, video embed, and featured service cards.
- Update the base `(infoPages)` layout so that it supports `TopTier` pages as well (hero image, photo credit, and title appear above the two-column layout).
  - The buttons on the Cards should vary depending on where they are in the order (first is `primary`, second is `secondary`, rest are `outline`).
- Some updates to the `Button` component:
  - Render a `Link` instead of a `button` if `isLink` is set to `true`.
  - Add a `secondary` variant (yellow).
  - Add some messy support for coloring our SVG icons that appear in `Button` components by setting the `filter` on these images so that they roughly match the `color`.
- Stub out a `VideoCard` component that just embeds a YouTube video in an `iframe` for now.

## Acceptance criteria validation

Following content should appear on the page:
- [x] Hero Image
- [x] Title
- [x] Subheading
- [x] Description
- [x] Featured service list (displayed as cards in a grid)

## Requested feedback

See self-review.

[LDAF-341]: https://ldaf.atlassian.net/browse/LDAF-341?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/routes/(infoPages)/[topTierPage]/+page.svelte
Comment (LouisFettet) [inline src/routes/(infoPages)/[topTierPage]/+page.svelte:42]: If y'all have any ideas on how to improve this let me know! The repetition here is quite ugly.

File: src/routes/(infoPages)/[topTierPage]/page.scss
Comment (LouisFettet) [inline src/routes/(infoPages)/[topTierPage]/page.scss:3]: Not sure if using the same `page.scss` file is the right approach, but it works for now. 

We no longer need this rule since we now handle icon colors properly (albeit messily) in the `Button` component styling.

**Review by LouisFettet — COMMENTED** [review]
self-review

File: src/routes/(infoPages)/[topTierPage]/+page.svelte
Comment (Benaiah) [inline src/routes/(infoPages)/[topTierPage]/+page.svelte:50]: We should replace this with the `Image` component eventually, but I don't think that needs to block this PR.

**Review by Benaiah — APPROVED** [review]

File: src/routes/(infoPages)/[topTierPage]/+page.svelte
Comment (LouisFettet) [reply src/routes/(infoPages)/[topTierPage]/+page.svelte:50]: Yeah, I figure once we merge your work with the blurhashes in Redis we can do a sweep and update all the images at once.

File: src/lib/components/Button/Button.svelte
Comment (hinzed1127) [reply src/lib/components/Button/Button.svelte:49]: I actually did the inverse quickly for service group pages [here](https://github.com/la-ldaf/ldaf-site/blob/main/src/routes/(infoPages)/%5BtopTierPage%5D/%5B...serviceGroupPage%5D/%2Bpage.svelte#L61-L63). It'd be good to choose one way and stick with it, but I'm down with having this option here, for now

File: src/routes/(infoPages)/[topTierPage]/+page.svelte
Comment (hinzed1127) [reply src/routes/(infoPages)/[topTierPage]/+page.svelte:42]: Unfortunately, I think this is the best solution until that support is added

File: src/routes/(infoPages)/[topTierPage]/+page.svelte
Comment (hinzed1127) [inline src/routes/(infoPages)/[topTierPage]/+page.svelte]: is this getting used anywhere?

**Review by hinzed1127 — APPROVED** [review]
small question on a style that might be unused, but looks good otherwise!

File: src/routes/(infoPages)/[topTierPage]/+page.svelte
Comment (LouisFettet) [reply src/routes/(infoPages)/[topTierPage]/+page.svelte]: Nope, noticed that when I was doing a self-review and removed it.

File: src/lib/components/Button/Button.svelte
Comment (LouisFettet) [reply src/lib/components/Button/Button.svelte:49]: Ah, yeah I might update that then to use my solution for now, just for consistency.

**Pull request merged**
